package.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Do you know why this is showing in the root of the project? I think this should only be in `integrations/vscode` or perhas the `.devcontainer` folder.

Reply:
During my debugging I had to manually install these packages due to missing deps; not sure how it got to the root folder but I moved it into integrations/vscode and re-tested. Looks to be working.